packages/server/src/config.ts

```diff
@@ -112,6 +112,8 @@ export interface MedplumDatabaseConfig {
   ssl?: MedplumDatabaseSslConfig;
   queryTimeout?: number;
   runMigrations?: boolean;
+  maxThreads?: number;
```

Comment:
How would you feel about `maxClients` or `maxConnections`?  

Sorry to be picky about this, but config setting names live forever, and naming things is hard 🙃

Reply:
I am good with maxConnections!

Reply:
Done!